prompt.txt

```diff
@@ -20,12 +20,12 @@ Do not creating invalid syntax
 Do not add unnecessary text in the response 
 Do not add notes or intro sentences 
 Do not show multiple distinct solutions to the question
-Do not add explanations on what the commands 
+Do not add explanations on what the commands do
 Do not return what the question was 
 Do not repeat or paraphrase the question in your response 
 Do not cause syntax errors
 Do not rush to a conclusion
 
 Follow all of the above rules. This is important you MUST follow the above rules. There are no exceptions to these rules. You must always follow them. No exceptions.
 
-Question: 
\ No newline at end of file
+Question: 
```

Comment:
In my opinion, since this the system message, this line is unnecessary as well.

Reply:
Thanks for the suggestion! Will merge it in.

Made me realize that there is bigger bug around the system message in the code. My intent was to make the first line of the prompt.txt the system message, and the user message is the entire contents of prompt.txt -  but right now it only takes one character for the system message - will fix this within the next hour.

Overall, regarding the system message. I'm a little unclear on what its for, but it seemed a good idea to have one.. The documentation says that the user message has stronger meaning, especially when it comes to instructions.

https://platform.openai.com/docs/guides/chat/instructing-chat-models:
"In general, gpt-3.5-turbo-0301 does not pay strong attention to the system message, and therefore important instructions are often better placed in a user message."